src/Testcontainers/Configurations/Modules/Databases/AzuriteTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,279 @@
+namespace DotNet.Testcontainers.Configurations.Modules.Databases
```

Comment:
It's `DotNet.Testcontainers.Configurations`

Reply:
Sorry, autogenerated vaule.
Fixed in [bddb8be](https://github.com/testcontainers/testcontainers-dotnet/pull/555/commits/bddb8be4dc91b51a89a704799bbe006d784e4f8c)